src/config/viewer.js

```diff
@@ -51,17 +68,11 @@ const PLUGINS_CONFIG = {
     merge: true,
     export: true
   },
-  properties: {
-    editProperties: true
-  },
+  tileset: true,
```

Comment:
you also have to add "tileset-viewer-parameters".
I think we want them enabled by default in viewer now